soh/include/variables.h

```diff
@@ -85,10 +85,10 @@ extern "C"
 	extern s16 gSpoilingItemReverts[3];
 	extern FlexSkeletonHeader* gPlayerSkelHeaders[2];
 	extern u8 gPlayerModelTypes[][5];
-	extern Gfx* D_80125DE8[];
-	extern Gfx* D_80125E08[];
-	extern Gfx* D_80125E18[];
-	extern Gfx* D_80125EF8[];
+	extern Gfx* sLeftHandWithTwoHandedSwordDLs[];
+	extern Gfx* sLeftHandDLs[];
+	extern Gfx* sLeftFistDLs[];
+	extern Gfx* sLeftHandWithBoomerangDLs[];
```

Comment:
```suggestion
	extern Gfx* gLeftHandWithTwoHandedSwordDLs[];
	extern Gfx* gLeftHandDLs[];
	extern Gfx* gLeftFistDLs[];
	extern Gfx* gLeftHandWithBoomerangDLs[];
```
just to make clear these ones are global. would have to change where theyre referenced too ofc

Reply:
Agreed `g` for global, `s` roughly means "static" as in static data stored locally in an actor's cpp file and only used for that actor.